data/api/server-server/events.yaml

```diff
@@ -137,6 +137,17 @@ paths:
                   type: string
                 example: ["$an_event:example.org"]
             required: ['auth_chain_ids', 'pdu_ids']
+        404:
+          description: |-
+            No event was found or the server doesn't know about the state at that event
+            to return anything useful.
+          examples:
+            application/json: {
+                "errcode": "M_NOT_FOUND",
+                "error": "Could not find event $Rqnc-F-dvnEYJTyHq_iKxU2bZ1CI92-kuZq3a5lr5Zg"
+              }
```

Comment:
I haven't actually experienced this response personally (because I'm not a server to observe but Synapse does error with these in the logs:

```
SynapseError: 404 - State not known at event $-04o8ieIyqp4uhGC1u3RbrW_991wYR3Tp3jqeVLUAAM
SynapseError: 404 - Could not find event $2k0rFR9Msehfk9bNCWo3eTvT1QQUwEgOXlVsQ_FNEhM
```